internal/rps/rps.go

```diff
@@ -132,7 +133,7 @@ func (amt *AMTActivationServer) ProcessMessage(message []byte) []byte {
 			log.Info(activation.Message)
 		} else {
 			log.Info("Status: " + statusMessage.Status)
-			log.Info("Network: " + statusMessage.Network)
+			log.Info("Network: " + strings.Replace(statusMessage.Network, "undefined. ", "", -1))
```

Comment:
"undefined" is coming from RPS right? If so, we should handle that on the RPS side, this seems a bit hack-ish.